grails-app/migrations/views/invoice-candidate-view.sql

```diff
@@ -38,8 +39,10 @@ union
         product.gl_account_id as gl_account_id,
         product.product_code as product_code,
         product.name as description,
-        shipment_item.quantity/order_item.quantity_per_uom as quantity,
-        shipment_item.quantity/order_item.quantity_per_uom as quantity_to_invoice,
+        case when invoice_item.id is not null then (shipment_item.quantity/order_item.quantity_per_uom) - invoice_item.quantity
```

Comment:
@dbienkowska is it possible, that the shipment item will have more than two invoice items? then it should be sum of all `invoice_item.quantity`

Reply:
@dbienkowska Additionally, do we need both `quantity` and `quantity_to_invoice`? Seems like these values are the same.